solution/ui/regulations/utilities/utils.test.js

```diff
@@ -80,18 +80,18 @@ describe("Utilities.js", () => {
         );
 
         const query2 = {
-            subjects: "1,2,asdfa",
+            subjects: ["1", "2", "asdf"],
             q: "test",
         };
 
         expect(getRequestParams(query2)).toBe("subjects=1&subjects=2&q=test");
 
-        const query3 = {
-            subjects: ["1,2", "3"],
-            q: "test",
-        };
+        //const query3 = {
```

Comment:
Any reason we are keeping the test instead of just removing it?  Not against it if its functionality we are bringing back that doesn't exist but will come back..